src/features/projectsV2/ProjectDetails/components/microComponents/SingleContactDetail.tsx

```diff
@@ -1,16 +1,20 @@
 import React from 'react';
-import styles from './ProjectInfo.module.scss';
-import RightArrowIcon from '../icons/RightArrowIcon';
+import styles from '../../styles/ProjectInfo.module.scss';
+import RightArrowIcon from '../../../../../../public/assets/images/icons/projectV2/RightArrowIcon';
 
 interface Props {
-  contactInfo: { icon: React.JSX.Element; title: string; link: string };
+  contactInfo: {
+    icon: React.JSX.Element;
+    title: string | null;
+    link: string | null;
+  };
 }
 
 const SingleContactDetail = ({ contactInfo }: Props) => {
   return (
     <a
-      href={contactInfo.link}
-      target="_blank"
+      href={contactInfo.link || undefined}
+      target={contactInfo.title === 'View Profile' ? '_self' : '_blank'}
```

Comment:
I'm not in favour of keeping logic specifically for an expected value of contactInfo.title.

Perhaps add another prop `shouldOpenNewTab` - default could be `false`, and we can explicitly pass `true` in when the link needs to open in another tab.